doc/release-notes/master.txt

```diff
@@ -10,6 +10,9 @@ Updates since v1.1.0
 
 TeleSculptor Application
 
+ * Removed deprecated command line maptk tools. All the underlying kwiver
+   algoritms can be accessed via the 'kwiver' command line tool.
```

Comment:
I would put this comment in a new section outside of "TeleSculptor Application".  Probably under a "Tools" heading.